src/payment/payment-methods/apple-pay/service.js

```diff
@@ -1,7 +1,19 @@
-import { App } from '@edx/frontend-base';
-import { logApiClientError } from '@edx/frontend-logging';
+import { ensureConfig, getConfig, mergeConfig } from '@edx/frontend-platform';
+import { logError } from '@edx/frontend-platform/logging';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
-App.ensureConfig([
+
+mergeConfig({
+  APPLE_PAY_MERCHANT_NAME: process.env.APPLE_PAY_MERCHANT_NAME,
```

Comment:
Isn't all of this merging done in index.jsx?

Reply:
(it only needs to be done once)

Reply:
yes, this like above is here to make tests function. I could add these lines into the test instead.

Reply:
config is now merged in setupTest